src/scss/rich-icons.scss

```diff
@@ -0,0 +1,283 @@
ule='nonzero' d='M152.451 67.554h-23.886v-.972h23.886v.486z'/%3e%3cpath id='Path' fill='%2333617E' fill-rule='nonzero' d='M152.451 71.734h-23.886v-.983h23.886v.486z'/%3e%3cpath id='Path' fill='%2333617E' fill-rule='nonzero' d='M152.451 75.903h-23.886v-.972h23.886v.486z'/%3e%3cpath id='Path' fill='%2333617E' fill-rule='nonzero' d='M152.451 80.084h-23.886V79.1h23.886v.497z'/%3e%3cpath id='Path' fill='%230099DA' fill-rule='nonzero' d='M136.742 80.462h-8.177V78.723h8.177v.875z'/%3e%3cpath id='Path' fill='%230099DA' fill-rule='nonzero' d='M136.742 76.282h-8.177v-1.729h8.177v.864z'/%3e%3cpath id='Path' fill='%23FFE700' fill-rule='nonzero' d='M136.742 72.112h-8.177V70.373h8.177v.864z'/%3e%3cpath id='Path' fill='%230099DA' fill-rule='nonzero' d='M136.742 67.932h-8.177v-1.728h8.177v.864z'/%3e%3cpath d='M128.565 100.456v2.074c0 .323.128.634.355.864.228.23.538.358.861.356h21.454c.323.002.633-.126.861-.356.227-.23.355-.54.355-.864v-2.074h-23.886z' id='Path' fill='%23194D6D' fill-rule='nonzero'/%3e%3cpath d='M151.709 106.224h-22.38a.526.526 0 0 1-.398-.173.639.639 0 0 1-.172-.432.606.606 0 0 1 .172-.432.505.505 0 0 1 .398-.184h22.38a.537.537 0 0 1 .409.184.65.65 0 0 1 .161.432.693.693 0 0 1-.161.432.57.57 0 0 1-.41.173z' id='Path' fill='%23CAD3D6' fill-rule='nonzero'/%3e%3cpath d='M151.235 61.138h-10.76v42.634h10.76c.323.002.633-.127.861-.357.227-.23.355-.54.355-.864V62.358a1.256 1.256 0 0 0-.355-.864 1.203 1.203 0 0 0-.86-.356z' id='Path' fill='%23D1D3D4' fill-rule='nonzero' style='mix-blend-mode:multiply'/%3e%3cpath class='animate-line' stroke='white' stroke-width='.5' fill='white' d='M83.45 41.154V57.25a3.019 3.019 0 0 1-3.012 3.024H43.651a3.04 3.04 0 0 1-3.023-3.024V37.2a3.474 3.474 0 0 0-1.008-2.47 3.447 3.447 0 0 0-2.457-1.019H17.377V27.76h-.463v12.962h.463v-6.556h19.786c.801 0 1.57.32 2.134.89a3.03 3.03 0 0 1 .879 2.145v20.048a3.493 3.493 0 0 0 3.475 3.489h36.787a3.468 3.468 0 0 0 2.464-1.018 3.496 3.496 0 0 0 1.022-2.471V41.154h-.473z' id='Path' fill-rule='nonzero'/%3e%3cpath id='Rectangle' fill='white' fill-rule='nonzero' d='M115.622 24.563h16.365v1h-16.365z'/%3e%3cpath class='animate-line-long' stroke='white' stroke-width='.5' fill='white' d='M164.47 118.149a3.106 3.106 0 0 1-.905 2.203 3.082 3.082 0 0 1-2.194.908h-20.734v-9.722h-.452v9.722H88.411a3.116 3.116 0 0 1-3.099-3.111v-6.211h-.452v6.21c0 1.971 1.588 3.57 3.55 3.576h72.982a3.533 3.533 0 0 0 2.518-1.044 3.561 3.561 0 0 0 1.033-2.531V55.985h-.452l-.021 62.164z' id='Path' fill-rule='nonzero'/%3e%3cpath class='animate-line' stroke='white' stroke-width='.5' fill='white' d='M77.694 114.012a2.08 2.08 0 0 1-2.076 2.074H18.485a2.05 2.05 0 0 1-1.471-.6c-.39-.39-.609-.921-.606-1.474v-5.023h-.452v5.023c-.006.674.26 1.322.735 1.797a2.491 2.491 0 0 0 1.794.73h57.133a2.513 2.513 0 0 0 1.787-.736 2.533 2.533 0 0 0 .741-1.791v-1.826h-.452v1.826z' id='Path' fill-rule='nonzero'/%3e%3cpath d='M119.947 104.776h-5.509V68.915c0-.613-.244-1.2-.678-1.631a2.32 2.32 0 0 0-1.625-.67H52.431a2.32 2.32 0 0 0-1.625.67 2.29
```

Comment:
this can be removed

Reply:
this file is in gitignore (I just removed it from git tracking)